parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -37,6 +37,8 @@ NETWORK_PLUGIN={{GetParameter "networkPlugin"}}
 NETWORK_POLICY={{GetParameter "networkPolicy"}}
 VNET_CNI_PLUGINS_URL={{GetParameter "vnetCniLinuxPluginsURL"}}
 CNI_PLUGINS_URL={{GetParameter "cniPluginsURL"}}
+## TODO parameterize this
+CRICTL_PLUGINS_URL="https://github.com/kubernetes-sigs/cri-tools/releases/download/v.17.0/crictl-v1.17.0-linux-amd64.tar.gz"
```

Comment:
cri-tools repo seems to doc a tight version matrix for versions. planning to fix this before merge?

Reply:
oh yea good catch. Gonna parameterize this. There's even a TODO for it above.

Reply:
TODO addressed